tests/test_chargebee_all_fields.py

```diff
@@ -0,0 +1,318 @@
+    fields_to_remove_common = {
+        'promotional_credits': {'amount_in_decimal'},
+        'invoices': {
+            'void_reason_code',
+            'expected_payment_date',
+            'voided_at',
+            'payment_owner',
+            'line_item_tiers',
+            'vat_number_prefix',
+            'total_in_local_currency',
+            'sub_total_in_local_currency',
+            'local_currency_code',
+            'next_retry_at'
+        },
+        'subscriptions': {
+            'create_pending_invoices',
+            'free_period',
+            'contract_term',
+            'plan_free_quantity_in_decimal',
+            'resume_date',
+            'override_relationship',
+            'auto_close_invoices',
+            'contract_term_billing_cycle_on_renewal',
+            'plan_amount_in_decimal',
+            'plan_quantity_in_decimal',
+            'has_scheduled_advance_invoices',
+            'free_period_unit',
+            'referral_info',
+            'pause_date',
+            'plan_unit_price_in_decimal',
+            'trial_end_action'
+        },
+        'customers': {
+            'vat_number_validated_time',
+            'referral_urls',
+            'offline_payment_method',
+            'entity_code',
+            'billing_day_of_week_mode',
+            'billing_date',
+            'use_default_hierarchy_settings',
+            'registered_for_gst',
+            'exemption_details',
+            'fraud_flag',
+            'exempt_number',
+            'vat_number_status',
+            'billing_day_of_week',
+            'parent_account_access',
+            'child_account_access',
+            'client_profile_id',
+            'is_location_valid',
+            'relationship',
+            'billing_date_mode',
+            'customer_type',
+            'mrr',
+            'auto_close_invoices',
+            'vat_number_prefix',
+            'business_customer_without_vat_number'
+        },
+        'credit_notes': {
+            'line_item_tiers',
+            'vat_number_prefix',
+            'total_in_local_currency',
+            'sub_total_in_local_currency',
+            'local_currency_code'
+        },
+        'payment_sources': {
+            'issuing_country',
+            'paypal',
+            'ip_address',
+            'bank_account',
+            'amazon_payment'
+        },
+        'transactions': {
+            'fraud_flag',
+            'authorization_reason',
+            'voided_at',
+            'reversal_txn_id',
+            'initiator_type',
+            'linked_payments',
+            'three_d_secure',
+            'merchant_reference_id',
+            'settled_at',
+            'reference_authorization_id',
+            'reversal_transaction_id',
+            'validated_at',
+            'fraud_reason',
+            'amount_capturable',
+            'reference_transaction_id'
+        },
+    }
+    # fields to remove for V2
+    fields_to_remove_V2 = {
```

Comment:
What is the reason behind skipping these fields? Do add comments for the same.

Reply:
I have added general comments for all the streams [here](https://github.com/singer-io/tap-chargebee/pull/83/files#diff-1f08d51aa03a23c2c59e9bec09f6da13b0bbede459fe4a8a3a5979e05ea79892R6).